src/components/ProductCard/ProductCard.tsx

```diff
@@ -22,6 +22,7 @@ const ProductCard = ({ product }: Props) => {
             src={product.images[0]}
             alt={product.name}
             width="250"
+            height="250"
```

Comment:
I'm gonna approve this but I want to also leave a comment:

This solution only works because all images from sephora API are squares. If the images had a different aspect-ratio, then they would look distorted. I hope you understand that, because in other situations this solution would not work.